lib/src/base-styles/_nx-btn.scss

```diff
@@ -135,9 +135,22 @@
 }
 
 .nx-btn--icon-only {
-  @extend .nx-btn--tertiary;
-
+  background-color: transparent;
   border-color: transparent;
+  color: var(--nx-color-text-dark);
+
+  &:hover {
+    background-color: var(--nx-swatch-grey-90);
```

Comment:
There's a variable for this: `--nx-color-interactive-background-hover`

Reply:
> There's a variable for this: `--nx-color-interactive-background-hover`

Excellent, much better: [cfef41f](https://github.com/sonatype/sonatype-react-shared-components/pull/990/commits/cfef41f493519717ac4aad50260c5ef22b4942fb)